src/cljs/main/broadfcui/common/icons.cljs

```diff
@@ -81,4 +81,4 @@
      {:size 28})
    :render
    (fn [{:keys [props]}]
-     (icon {:style {:color "white" :fontSize (:size props)}} :warning-triangle))})
+     (icon {:style (merge {:color "white" :fontSize (:size props)} props)} :warning-triangle))})
```

Comment:
I suspect this may not be what you really want to do. Why merge this in if `:size` is already being pulled from it?